### PR TITLE
fix/56: replace modal image and fix styles

### LINKED ISSUE #56: Update And Check The Arrived Modal
### Explanation:
Arrival confirmation window, once the map generates that state the confirmation window appears, styles and existing image, and button styles must be changed, organize the content of the view that looks good in different mobile resolutions, take into account spaces and margins in the design.

A design link in figma is attached where you can also have access to graphic resources.

### SCREENSHOTS
image

### CHECKLIST 

- [ ] Change the color of the "accept" button to a blue color with code `#005D9D`
- [ ] Change Delivery image.
- [ ] Transparency wallpaper for popup window #0000FF opacity at 17%.

### REFERENCES
https://www.figma.com/file/tbhMgDT10bbVjB9GazF97y/Dasher?node-id=420%3A249
https://www.figma.com/file/tbhMgDT10bbVjB9GazF97y/Dasher?node-id=522%3A28

### PR DESCRIPTION
**Pull request by Eghinner**
### Description

Mainly replaced the arrived image.
Added custom global styles in the Text component.
Used a div container for the texts to prevent global styles from affecting them
Fixed the opacity of the modal component and some adjustments in the Modal.styled file for passing linters

fixes #56

#### Screenshots

localhost_3000_delivery_b0e71480-77f0-4b61-83df-bfe4c4a0c47d(Nest Hub Max)
localhost_3000_delivery_b0e71480-77f0-4b61-83df-bfe4c4a0c47d(BlackBerry Z30)

### How to test it

1. Go to '/delivery/b0e71480-77f0-4b61-83df-bfe4c4a0c47d'

### Checklist:

- [x] Passes linters and test, also is building.
- [x] Doesn't have spelling or grammatical problems.
- [x] Doesn't have unnecessary comments or debugging code.
- [x] The branch is updated with the last dev branch changes.

### CODE REVIEW

File: src/components/Text.jsx [hunk 1]
```diff
@@ -73,6 +86,18 @@ const Text =
           font-size: 1rem;
         }
       `}
+
+  ${
+  /** @param {TextProps} props */
+  props =>
+      props.mlg &&
```
Comment (Israel-Laguan) [inline src/components/Text.jsx]: Hmm `xsm` sounds to me like "xtra-small", but what means `mlg`? Like "medium-large"? What about you use something more descriptive, for example "large". I think is better to keep it clear, even if it uses a lot of characters

File: src/pages/Delivery/index.jsx [hunk 2]
```diff
@@ -68,13 +68,13 @@ const Delivery = () => {
           }}
           actionText="Aceptar"
         >
-          <Text as="h1" color="primary" medium center>
-            El Dasher ha llegado
+          <Text as="div" margin="5" color="six" mlg weight700 center>
```
Comment (Israel-Laguan) [inline src/pages/Delivery/index.jsx]: The value for this color seems like is not correct, please verify

Comment (Eghinner) [reply src/pages/Delivery/index.jsx]: That's right. I have verified

File: src/components/Text.jsx (hunk 1 above)
Comment (Eghinner) [reply src/components/Text.jsx]: `large` already exists, and also `lg`.
I was checking and only `NotFound` page has this prop. I could change it to `xlarge` which is a bit bigger (from `2.25rem` to `3rem`)
image

**Comment by Israel-Laguan** [comment]
Please review with latest dev, linter issues fixed

File: src/components/Text.jsx (hunk 1 above)
Comment (heliomar-pena) [reply src/components/Text.jsx]: > `large` already exists, and also `lg`. I was checking and only `NotFound` page has this prop. I could change it to `xlarge` which is a bit bigger (from `2.25rem` to `3rem`) 

I think it's good, I think that looks better with 3rem, what do you think? @Israel-Laguan @angelik0828

File: src/pages/Delivery/index.jsx (hunk 2 above)
Comment (heliomar-pena) [reply src/pages/Delivery/index.jsx]: Hello, the color that is in figma is: `#0000FF` with opacity 17%

imagen

And the actual color in the app is: `#005D9D`

Is that what you meant @Israel-Laguan?

File: src/components/Text.jsx (hunk 1 above)
Comment (Israel-Laguan) [reply src/components/Text.jsx]: Looks good to me

File: src/pages/Delivery/index.jsx (hunk 2 above)
Comment (Israel-Laguan) [reply src/pages/Delivery/index.jsx]: I mean, is there a color called `six`? Also not sure if there is a prop named `weight700`?

Comment (Eghinner) [reply src/pages/Delivery/index.jsx]: Well, I can replace `weight700` for `bolder`. And I think `color="six"` for `blue`.

Comment (Israel-Laguan) [reply src/pages/Delivery/index.jsx]: That sounds better!

Comment (Israel-Laguan) [reply src/pages/Delivery/index.jsx]: @e-azocar what do you think of the naming?

File: src/components/Text.jsx (hunk 1 above)
Comment (Eghinner) [reply src/components/Text.jsx]: done

**Review by Israel-Laguan — APPROVED** [review]

**Review by e-azocar — APPROVED** [review]
Good job

File: src/pages/Delivery/index.jsx (hunk 2 above)
Comment (Eghinner) [reply src/pages/Delivery/index.jsx]: > Well, I can replace weight700 for bolder. And I think color="six" for blue.


Done

**Review by Israel-Laguan — APPROVED** [review]

**Comment by heliomar-pena** [comment]
Please resolve conflicts

File: src/components/Text.jsx [hunk 3]
```diff
@@ -63,6 +64,17 @@ const Text =
...
+  ${
+  /** @param {TextProps} props */
+  props =>
+      props.xsm &&
+      css`
+        ${text.xl};
+        @media (max-width: 960px) {
+          ${text.sm};
+        }
+      `}
```
Comment (Katsu08) [inline src/components/Text.jsx:76]: I understand that xsm is: `sm` on small screens and `xl` on big screens, but I agree with @Israel-Laguan, xsm sounds like "eXtra SMall"

**Review by heliomar-pena — APPROVED** [review]
Looks so good! Good work! Please check conflicts

File: src/pages/Delivery/index.jsx [hunk 4]
```diff
@@ -43,20 +43,12 @@ const Delivery = () => {
...
         ?
         <Modal
-          isOpen={currentStatus === 'destination_reached' && openDeliveryConfirmedModal}
-          handleClick={() => {
-            toggleDeliveryConfirmedModal(!openDeliveryConfirmedModal)
-            toConfirm()
-          }}
-          actionText="Aceptar"
+          isOpen={true}
+          handleClick={() => { history.push('/check') }}
```
Comment (Israel-Laguan) [inline src/pages/Delivery/index.jsx:47]: @Katsu08 please check if removing the other actions are correct

Comment (heliomar-pena) [reply src/pages/Delivery/index.jsx:47]: I'm checking

Comment (heliomar-pena) [reply src/pages/Delivery/index.jsx:47]: > @Katsu08 please check if removing the other actions are correct

They wasn't removed, they still are here: 

https://github.com/Azordev/maps-project/blob/d6abd0af2590873fcaaebe5844fe88f63489d314/src/pages/Delivery/index.jsx#L55-L70

@Israel-Laguan

**Review by heliomar-pena — APPROVED** [review]
Good work!

**Pull request merged**